contents/blog/clickhouse-vs-snowflake.md

```diff
@@ -21,105 +21,111 @@ If I was to compare the databases to boats, ClickHouse is a jet ski – ultra-fa
+- A dedicated non-technical-friendly GUI for exploring and visualizing data
+- The ability to modify entry data without massive performance burns
+- The ability to scale architecture seamlessly with no migrations whatsoever
 
-### Hosted ClickHouse
+### ClickHouse Cloud and Altinity Cloud
 
-It is slightly silly to compare Snowflake and ClickHouse without mentioning ClickHouse Cloud, Altinity Cloud, Firebolt, or TinyBird. Each solution simplifies deploying and maintaining a ClickHouse instance. This helps address the final bullet in the previous section; they make scaling architecture seamless without complex data migrations.
+It is slightly silly to compare Snowflake and ClickHouse without mentioning ClickHouse Cloud and Altinity Cloud. Both solutions simplify deploying and maintaining a ClickHouse instance. This helps address the final bullet in the previous section; they make scaling architecture seamless without frustrating data migrations.
 
-ClickHouse Cloud – a new product launched in 2022 by ClickHouse Inc (ClickHouse’s primary contributor) – is exclusively available on AWS, with plans to support both Google Cloud and Azure like Snowflake. ClickHouse Cloud is engineered like Google BigQuery and Snowflake. The data is stored in object storage and you can scale up compute nodes on demand. This makes capacity planing and scaling a breeze. You never have to worry about resharding or worrying about capacity planning for the number of disks in your cluster (*swoon*). ClickHouse also purchased Arctype to provide a top tier GUI for exploring and visualizing data.
+ClickHouse Cloud – a new product launched in 2022 by ClickHouse Inc (ClickHouse’s primary contributor) – is exclusively available on AWS, with plans to support both Google Cloud and Azure like Snowflake. Meanwhile, Altinity Cloud – built by Altinity Inc (also contributors to ClickHouse) – has support for both AWS and Google Cloud. Altinity Cloud is also Kubernetes-compatible. 
 
-Altinity Cloud – built by Altinity Inc (also contributors to ClickHouse) – has support for both AWS and Google Cloud. Altinity Cloud is also Kubernetes-compatible.  Altinity Cloud runs ClickHouse similar to how an on-prem or local instance works; virtually anything you can do on a local instance is possible on Altinity Cloud. 
+The big difference between ClickHouse Cloud and Altinity Cloud is how they store data. ClickHouse Cloud is engineered like Google BigQuery. The data is stored in object storage. Altinity Cloud meanwhile runs ClickHouse similar to how an on-prem or local instance works; virtually anything you can do on a local instance is possible on Altinity Cloud. 
```

Comment:
Unless I'm missing something, most of these changes are marketing language rather than factual. If so, we should stick with the original copy.